src/SortableContainer/index.js

```diff
@@ -339,12 +341,21 @@ export default function SortableContainer(WrappedComponent, config = {withRef: f
 				if (transitionDuration) {
 					node.style[`${vendorPrefix}TransitionDuration`] = `${transitionDuration}ms`;
 				}
+
+				const margin = getElementMargin(node);
+				// margins merge so we subtract them
+				const delta = (
+					axis === 'x'
+					? this.width  + this.margin.left + this.margin.right  - margin.left
+					: this.height + this.margin.top  + this.margin.bottom - margin.top
+				);
+
```

Comment:
@clauderic 
I found that margin collapsing is a very hard to solve problem and a rarely used feature.
So I didn't add support for it.
This PR is now ready for merging.